buildqtc.sh

```diff
@@ -428,6 +428,17 @@ build_windows_qtc() {
         curl -s -f -O http://$OPT_UPLOAD_HOST/sailfishos/win32-binary-artifacts/pkg-config/$lib
     done
 
+    if [[ $DEF_MSVC_VER -lt 2019 ]]; then
+        VCVARSALL="%_programs%\microsoft visual studio $DEF_MSVC_VER_ALT\vc\vcvarsall.bat"
+        D3DCOMPILER="%_programs%\microsoft visual studio $DEF_MSVC_VER_ALT\vc\bin\D3Dcompiler_*.dll"
+        VCREDISTCRT="%_programs%\microsoft visual studio $DEF_MSVC_VER_ALT\vc\redist\x86\microsoft.vc*.crt"
+    else
+        VCVARSALL="%_programs%\Microsoft Visual Studio\\$DEF_MSVC_VER\BuildTools\VC\Auxiliary\Build\vcvarsall.bat"
+        VCVARSALL_PARAMS="x86"
+        D3DCOMPILER="%VCToolsInstallDir%\bin\Hostx86\x86\d3dcompiler_*.dll"
+        VCREDISTCRT="%VCToolsRedistDir%\x86\Microsoft.VC*.CRT"
+    fi
+
```

Comment:
I would prefer simply updating the literal values for the version we currently use. Should someone need to use other version, they should do the required modifications in their own Git branch.